images/scripts/fedora.setup

```diff
@@ -175,11 +175,19 @@ fi
 dnf $DNF_OPTS -y upgrade
 dnf $DNF_OPTS -y install $ALL_DEPS
 
+# create/run this as a script, so that it can be re-run at image-prepare time with different repos
+cat > /var/lib/download-package-sets.sh <<EOF
+#!/bin/sh
+set -eu
+rm -rf /var/lib/download-package-sets
```

Comment:
This should be `rm -rf /var/lib/package-sets`, no?

Reply:
Right, thanks! Fixed